CHANGELOG.md

```diff
@@ -65,34 +68,26 @@ These should not affect the functionality of the module.
 
 **Merged pull requests:**
 
+- Release 1.1.0 [\#219](https://github.com/voxpupuli/puppet-selinux/pull/219) ([oranenj](https://github.com/oranenj))
 - Fedora 26 uses the same package\_name as Fedora 25 [\#218](https://github.com/voxpupuli/puppet-selinux/pull/218) ([logic](https://github.com/logic))
 - Ensure the module build tmp/ directory is actually a directory [\#216](https://github.com/voxpupuli/puppet-selinux/pull/216) ([oranenj](https://github.com/oranenj))
-- Prepare 1.0.0 [\#211](https://github.com/voxpupuli/puppet-selinux/pull/211) ([oranenj](https://github.com/oranenj))
 
 ## [v1.0.0](https://github.com/voxpupuli/puppet-selinux/tree/v1.0.0) (2017-04-02)
-
 [Full Changelog](https://github.com/voxpupuli/puppet-selinux/compare/v0.8.0...v1.0.0)
 
-**Breaking changes:**
```

Comment:
Can you please generate this with the git version of github-changelog-generator? this 'breaking changes section thing' isn't included in the last release :(

Reply:
regenerated with git master branch of github-changelog-generator